### DISCUSSION TITLE
Information to solve problem

### DISCUSSION THREAD

**Discussion by Mcezarf** [TfTHacker/DashboardPlusPlus#2] [Q&A]
see the text below copied from ++Dashboard. Is this code run by .css? The result does not show as it is in the article. What needs to be done? the commands are shown with the figure!!!! Apparently the banner and div etc commands are not being recognized.

---
cssclass: dashboard
---

banner: "![[02 - Anexos/home.jpg]]"
banner_y: 0.55
banner_x: 0.5

<div class="title" style="color:Sienna">HOME</div>
---

**Comment by TfTHacker** [comment]
The HOME should be below the ---

Look at the file in raw mode, might be clearer the text flow:

https://raw.githubusercontent.com/TfTHacker/DashboardPlusPlus/master/Dashboard%2B%2B.md

**Reply by Mcezarf** [reply]
I did this but It does not working. See the screen attached

![Screen in edition mode](https://user-images.githubusercontent.com/88067013/160404017-e8ec9d27-f0e2-46e9-a396-ab2bb95e3b80.jpg)
![screen in view mode](https://user-images.githubusercontent.com/88067013/160404044-6b303482-bb54-46a0-b03e-6ac08662fb0d.jpg)
.

**Reply by TfTHacker** [reply]
This looks like the CSS for dashboard is not installed or enabled. Can you confirm this?

**Reply by Mcezarf** [reply]
See the screen file attached
![Configuration CSS](https://user-images.githubusercontent.com/88067013/161281996-f7801707-93cb-4913-8af1-77aa486ef379.jpg)
![Files in snippets](https://user-images.githubusercontent.com/88067013/161282014-d3d827d1-9e0a-4950-8ba0-28d5c04b4cac.jpg)
![console](https://user-images.githubusercontent.com/88067013/161282031-f7dbedaa-603f-4e80-83cd-226449980ad3.jpg)
d

**Comment by sirlaughalot90** [comment]
Hello, thank you for the great work! I can't get it to work quite right:

<img width="806" alt="CleanShot 2022-03-25 at 09 14 12@2x" src="https://user-images.githubusercontent.com/20352339/160081541-493a0623-b144-4959-b9ee-00d8f07c13f8.png">

I use Minimal Theme with Image Grids. Disabling Contextual Typography didn't solve it. Do you have any ideas on what setting to fiddle with? I am almost certain, it is just some setting in Minimal Theme. I copied some of your examples in order to check if I have done the formatting wrong but it makes no difference.

**Reply by TfTHacker** [reply]
Is this the full width of the screen?

**Reply by sirlaughalot90** [reply]
Well, there is empty space on both sides and the two sidebars. But there are no more columns if this is what you mean:
![CleanShot 2022-03-28 at 22 55 04@2x](https://user-images.githubusercontent.com/20352339/160485689-562fb78c-af8a-4dcf-b3a7-a4d7a7767dae.png)

**Reply by TfTHacker** [reply]
I see. You need to adjust the "readable line length" as mentioned in the article.

I also added to this repo another css to force this setting on dashboard pages:

https://github.com/TfTHacker/DashboardPlusPlus/blob/master/.obsidian/snippets/dashboard-ReadLineLength.css

**Reply by sirlaughalot90** [reply]
This solves it... kind of:

<img width="1239" alt="CleanShot 2022-03-29 at 11 47 20@2x" src="https://user-images.githubusercontent.com/20352339/160584455-3af21e80-cd66-43ba-82b5-0e8044213b6e.png">

The fourth column is still being displayed below.

**Reply by sirlaughalot90** [reply]
Plus, when I open the sidbars it goes back to only 2 columns being displayed. I would guess that your approach only works on big displays with enough vertical space, not on a MB Air. I'll test it later today when I have access to a larger display.

**Reply by TfTHacker** [reply]
I can't say I have very large display. But of course each device can be different. Basically as you see in your picture the sections wrap to new rows based on size.

You will see in the dashboard.css file:
```css
.dashboard div > ul > li {
    min-width: 250px;
    width: 15%;
}
```

You can experiment with making min-width smaller by changing 250 to lets say 200 and so on.

I also personally added this to reduce padding more:
```css
.dashboard .markdown-preview-section {
    padding-left:  10px !important;
    padding-right: 10px !important;
}
```

Let me know if one of these strategies worked.

**Reply by sirlaughalot90** [reply]
Thank you. I did some testing, and it basically comes down to display size and readable line length. RLL makes the biggest difference. With RLL enabled I always only get 2 columns, no matter how big the screen. With RLL disabled, I get three columns on my MacBook and all four when plugged into my 4k display.  My main problem is that I really do like RLL. 😅 Probably, I'll stick to RLL and use your (very nice) CSS with only two columns. 😇

**Reply by TfTHacker** [reply]
That is progress then. But I have a css file in the repo for overriding readline setting. The article mentions this.
